Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -2584,3 +2592,30 @@ int CLuaElementDefs::IsElementWaitingForGroundToLoad(lua_State* luaVM)
     lua_pushboolean(luaVM, false);
     return 1;
 }
+
+int CLuaElementDefs::GetElementCount(lua_State* luaVM)
+{
+    // Verify the argument
+    EElementCountType eType;
+    CScriptArgReader  argStream(luaVM);
+    argStream.ReadEnumString(eType);
+
+    if (!argStream.HasErrors())
+    {
+        // Find our VM
+        CLuaMain* pLuaMain = m_pLuaManager->GetVirtualMachine(luaVM);
+        if (pLuaMain)
+        {
+            size_t iCount = CStaticFunctionDefinitions::GetElementCount(eType);
+            lua_pushnumber(luaVM, iCount);
+            return 1;
+        }
+    }
+
+    if (argStream.HasErrors())
+        m_pScriptDebugging->LogCustom(luaVM, argStream.GetFullErrorMessage());
+
+    // Failed
+    lua_pushboolean(luaVM, false);
+    return 1;
+}
```

Comment:
Use the new parser. See other PRs for examples. (Search for ArgumentParser in CLuaDefs)